introduction/differences.md

```diff
@@ -48,6 +48,7 @@ The most noteworthy changes since the previous release are described here.
         * The cray-dns-unbound-manager CronJob will be deprecated in a future release once all DNS records are migrated to PowerDNS.
         * The introduction of PowerDNS and Bifurcated CAN will introduce some node and service naming changes.
         * Please see the [PowerDNS migration notice](../introduction/PowerDNS_migration.md) for more information.
+   * The Compute Rolling Upgrade Service (CRUS) will be deprecated in the CSM-1.3.0 release. Enhanced BOS functionality will replace CRUS. This includes the ability to stage changes to nodes that can be acted upon later when the node reboots. It also includes the ability to reboot nodes without specifying any boot artifacts. This latter ability relies on the artifacts already having been staged.
```

Comment:
This doesn't seem very emphasized for something that seems like an important note for admins.
Perhaps prefixing it with 
```markdown
**`NOTE`**
```
Or bolding the first sentence would do it more justice?
![Screenshot_20211130-122355.png](https://user-images.githubusercontent.com/7772179/144105635-0a526b1e-341e-471d-b124-b5edd6f90390.png)

Reply:
> 

@rustydb , are you talking about my addition needing more emphasis or the entire section as a whole? I just followed the established pattern when I added mine.

Reply:
The addition.

Reply:
_looks again at the other contents in the file_

I suppose there are other deprecations in here already, so maybe this is totally moot.